src/edu/csus/ecs/pc2/AppConstants.java

```diff
@@ -36,6 +36,8 @@ public class AppConstants {
     public static final String NO_SAVE_OPTION_STRING = "--nosave";
 
     public static final String PASSWORD_OPTION_STRING = "--password";
+    
```

Comment:
Can we move this new option so it keeps the list of options alphabetical?  Right before --contestpassword.

Reply:
I do not think that such a change is required.    With the invention of searching the code
finding constants does not require any particular order.    I do not see any significant value in alphabetizing
constants and think that is extra unnecessary work.   It also may require future, again, unnecessary merge work.

Reply:
pushed fix.